src/components/tree-views-container.js

```diff
@@ -728,15 +730,33 @@ const TreeViewsContainer = () => {
                 error={renameError}
             />
             <DeleteDialog
-                message={intl.formatMessage({
-                    id: 'deleteDirectoryDialogMessage',
-                })}
+                message={
+                    <span>
+                        {intl.formatMessage(
+                            {
+                                id: 'deleteDirectoryDialogMessage',
+                            },
+                            {
+                                directoryName: (
+                                    <span style={{ fontWeight: 'bold' }}>
+                                        {getTargetDirectory()}
+                                    </span>
+                                ),
+                            }
+                        )}
+                    </span>
+                }
                 open={openDeleteDirectoryDialog}
                 onClick={deleteSelectedDirectory}
                 onClose={handleCloseDeleteDirectoryDialog}
-                title={intl.formatMessage({
-                    id: 'deleteDirectoryDialogTitle',
-                })}
+                title={intl.formatMessage(
```

Comment:
must be managed inside the deleteDialog component not in tree-views-container

Reply:
done